cobra/flux_analysis/sampling.py

```diff
@@ -56,6 +56,40 @@
 """
 
 
+def mp_init(obj):
+    """Initialize the multiprocessing pool."""
+    global sampler
```

Comment:
Only needed for OptGP and I didn't want to pollute the module name space. But no strong feelings either way so whatever you prefer.